modules/integration/tests-integration/src/test/java/org/wso2/das/integration/tests/clustering/MinimumHAClusterTestCase.java

```diff
@@ -0,0 +1,357 @@
+            this.dasServerManagers.put(instanceName, serverManager);
+
+            if (i == 0) {
+                firstCarbonHome = this.startServer(instanceName);
+            }
+            i++;
+        }
+        return firstCarbonHome;
+    }
+
+    private String startServer(String instanceName)
+            throws XPathExpressionException, AutomationFrameworkException, IOException, AutomationUtilException {
+        log.info("#####################################  Starting server : " + instanceName);
+        String carbonHome = this.dasServerManagers.get(instanceName).startServer();
+
+        AutomationContext context = this.dasServerManagers.get(instanceName).getContext();
+        ConfigurationContext configContext = ConfigurationContextFactory.createConfigurationContextFromFileSystem(null);
+        String loggedInSessionCookie = new LoginLogoutClient(context).login();
+        AnalyticsProcessorAdminServiceStub analyticsStub = new AnalyticsProcessorAdminServiceStub(configContext,
+                                                                                                  context.getContextUrls().getBackEndUrl() + "/services/" + ANALYTICS_SERVICE_NAME);
+        ServiceClient client = analyticsStub._getServiceClient();
+        Options option = client.getOptions();
+        option.setManageSession(true);
+        option.setProperty(org.apache.axis2.transport.http.HTTPConstants.COOKIE_STRING,
+                           loggedInSessionCookie);
+        this.analyticsStubs.put(instanceName, analyticsStub);
+
+        return carbonHome;
+    }
+
+    private void stopServer(String instanceName)
+            throws XPathExpressionException, AutomationFrameworkException, IOException {
+        log.info("##################################### Stopping server : " + instanceName);
+        dasServerManagers.get(instanceName).stopServer();
+    }
+
+
+    @Test(groups = "wso2.das.clustering", description = "Starting master2")
+    public void clusterInitialization() throws Exception {
+        log.info("##################################### cluster Initialization");
+        startServer("master2");
+        runScriptTest("master1");
+    }
+
+    @Test(groups = "wso2.das.clustering", description = "Stopping master2", dependsOnMethods = "clusterInitialization")
+    public void shutdownMaster2() throws Exception {
+        log.info("##################################### shutdown Master2");
+        stopServer("master2");
+        runScriptTest("master1");
+    }
+
+    @Test(groups = "wso2.das.clustering", description = "Stopping master2", dependsOnMethods = "shutdownMaster2")
+    public void restartMaster2() throws Exception {
+        log.info("##################################### restart Master2");
+        startServer("master2");
+        runScriptTest("master1");
+    }
+
+    private void runScriptTest(String activeMasterInstanceName) throws Exception {
```

Comment:
The @Test annotation has not been added to this test method, hence it will not run.